table/block_based/filter_policy.cc

```diff
@@ -61,6 +66,20 @@ class XXPH3FilterBitsBuilder : public BuiltinFilterBitsBuilder {
     // requirements.
     if (hash_entries_.empty() || hash != hash_entries_.back()) {
       hash_entries_.push_back(hash);
+      if (cache_res_mgr_ &&
+          (hash_entries_.size() % (kUint64tHashEntryCacheResBucketSize / 2)) ==
```

Comment:
I think we can simplify:

    if (cache_res_mgr_ &&
        // Traditional rounding to whole bucket size
        (hash_entries_.size() % kUint64tHashEntryCacheResBucketSize) == kUint64tHashEntryCacheResBucketSize / 2) {